docs/gatsby/api-reference/ocular-config.md

```diff
@@ -48,7 +48,7 @@ Each link entry has the following fields:
 
 | Field     | Type     | Name  |
 | ---       | ---      | ---   |
-| `label`    | `String` | Required. The displayed text in the header link. |
+| `name`    | `String` | Required. The displayed text in the header link. |
 | `href`    | `String` | Optional. The hyperlink you want to redirect to. |
 | `to`   | `String` | Optional. The path to an internal gatsby page. Each entry will either only has `href` or `to`. |
```

Comment:
"internal gatsby page" = one of the pages generated by ocular-gatsby?

Ideally, at some point, we should be able to predict that based on URL.